index.js

```diff
@@ -22,7 +22,9 @@ module.exports = function (input) {
           let sentence = node.children
             .filter((c) => ['WordNode', 'PunctuationNode', 'SymbolNode'].includes(c.type))
             .map((c) => toString(c));
-          let tags = new Tag(sentence).initial().smooth().tags;
+
+            // HACK: Truncate any word longer than 40 chars as en-pos will not be performant.
+          let tags = new Tag(sentence.map((s) => s.slice(0, 40))).initial().smooth().tags;
```

Comment:
I would probably replace `s` with `word` or `node` or `w` or `n`. `s` is a little confusing.